examples/README.md

```diff
@@ -0,0 +1,74 @@
+# Description
+
+This is a showcase of resources that can be provisioned using the Ably Terraform provider.
+Information on the Ably Control API can be found here - [Ably Control API Docs](https://ably.com/docs/api/control-api)
+
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_ably"></a> [ably](#provider\_ably) | n/a |
```

Comment:
What's with the <a> tag here?

Reply:
terraform-docs creates anchor links by default (https://terraform-docs.io/reference/markdown-document/)

Reply:
Ah right this was auto generated.